scan/src/components/ChainInfoHighlights.re

```diff
@@ -50,146 +50,147 @@ module Styles = {
 
 module HighlightCard = {
   [@react.component]
-  let make = (~label, ~valueComponent, ~extraComponent, ~extraTopRight=?, ~bgUrl=?) => {
+  let make = (~label, ~valueAndExtraComponentSub: ApolloHooks.Subscription.variant(_), ~bgUrl=?) => {
     <div className=Styles.card>
-      {switch (bgUrl) {
-       | Some(url) => <div className={Styles.bgCard(url)} />
-       | None => React.null
+      {switch (bgUrl, valueAndExtraComponentSub) {
+       | (Some(url), Data(_)) => <div className={Styles.bgCard(url)} />
+       | _ => React.null
        }}
       <div className=Styles.innerCard>
         <div className=Styles.labelContainer>
-          <Text value=label color=Colors.bandBlue spacing={Text.Em(0.05)} />
-          {extraTopRight->Belt.Option.getWithDefault(React.null)}
+          {switch (valueAndExtraComponentSub) {
+           | Data(_) => <Text value=label color=Colors.bandBlue spacing={Text.Em(0.05)} />
+           | _ => <LoadingCensorBar width=90 height=18 />
+           }}
         </div>
-        valueComponent
-        <div className={Styles.withWidth(170)}> extraComponent </div>
+        {switch (valueAndExtraComponentSub) {
+         | Data((valueComponent, extraComponent)) => <> valueComponent extraComponent </>
+         | _ =>
+           <> <LoadingCensorBar width=120 height=20 /> <LoadingCensorBar width=75 height=15 /> </>
+         }}
```

Comment:
IMO, we can put the switch `valueAndExtraComponentSub ` together and then switch on `bgUrl` inside.

Reply:
However, I think it's ok for now.